requirements.txt

```diff
@@ -2,9 +2,9 @@ black==21.12b0
 flake8==3.8.4
 iso639==0.1.4
 mypy==0.790
-pytest==6.1.1
+pytest==7.0.0
```

Comment:
This update is needed to resolve the Python 3.10 build failure at 47795e91dee1a41af20186014f8dc2ccee128a36